FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -30,7 +30,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !UserDefaults.standard.bool(forKey: UserDefaultsKeys.hasShownWelcomeScreen.rawValue) {
             showWelcome()
         }
-        
+
+        var types = UIUserNotificationType()
+        types.insert(.alert)
+        types.insert(.badge)
+        UIApplication.shared.registerUserNotificationSettings(UIUserNotificationSettings(types: types, categories: nil))
```

Comment:
1. Agreed about needing ios9 support. that was sloppy.
2. Agreed, notifications should be off by default and not requested until user acts as you suggest. I forget when building these system permissions thing because it only asks me once.
3. I like the idea of presenting it to them early, but an alternative might also be to wait and present a toolTip on the **2nd** time they have used the app. It might be less likely to bother lookie-loo users concerned about privacy, and more likely to be accepted by users demonstrating a minimal commitment to using the app? I don't think that is an overly complex logic.
4. I assume that this we would still keep the entry in the `aboutViewController`.

Perhaps the best way forward would be to move the `UIApplication.shared.registerUserNotificationSettings()` call to the `ScheduleRemindersController` and trigger it on first user action in that screen.

then merge that, and put the rest of these items into a new "issue". I would be fine with someone else doing those as well. But whatever you guys think best.

@mccarron , @subdigital